src/app/navbar/navbar.component.ts

```diff
@@ -1,17 +1,31 @@
-import { Component, Input } from '@angular/core';
+import { Component, Input, OnInit } from '@angular/core';
 
 import { AuthService } from '../auth/auth.service';
 
+import { apiHost } from '../constants';
+
 
 @Component({
   selector: 'ccl-navbar',
   templateUrl: './navbar.component.html'
 })
-export class NavbarComponent {
+export class NavbarComponent implements OnInit {
 
     @Input() projectID: string;
 
     public isSettingsDropdownOpen = false;
+    public apiHost = '';
 
     constructor(public authService: AuthService) {}
+
+    ngOnInit() {
+      const lowercaseApiHost = apiHost.toLowerCase();
+      if (lowercaseApiHost.indexOf('localhost') !== -1) {
+        this.apiHost = 'localhost:8080';
```

Comment:
Rather than take the apiHost value as is, is this if/else block for consistency?

Reply:
Oh my god. I have no excuse, idk what I was doing there. Yes, this should just take the apiHost value directly.

Reply:
haha oops